test/test_python.py

```diff
@@ -422,6 +422,22 @@ def get_value(self, value):
     assert selenium.run_js("return pyodide.pyimport('f').toString()").startswith("<Foo")
 
 
+def test_pyproxy_refcount(selenium):
+    selenium.run(
+        """
+        import sys
+        from js import document
+
+        def pytest(*args, **kwargs):
+            print(*args, **kwargs)
+
+        document.addEventListener("click", pytest)
+        document.removeEventListener("click", pytest)
+        """
+    )
+    assert selenium.run("sys.getrefcount(pytest)") == 3
+
+
```

Comment:
Would another few calls to check that the reference count doesn't increase be helpful? 
```suggestion
    selenium.run("""
        document.addEventListener("click", pytest)
        document.removeEventListener("click", pytest)
        """
    )
    # check that the reference count doesn't increase when calling python2js repeatedly
    assert selenium.run("sys.getrefcount(pytest)") == 3          
```

Also could we call that function `func` or some other generic name, as the name `pytest` can lead to confusion.

Reply:
Hi @rth, I've just replaced the example by a more detailed version, and with your suggestions.